kedro/framework/cli/project.py

```diff
@@ -147,17 +149,18 @@ def ipython(metadata: ProjectMetadata, env, args, **kwargs):  # noqa: unused-arg
 @click.pass_obj  # this will pass the metadata as first argument
 def package(metadata: ProjectMetadata):
     """Package the project as a Python wheel."""
-    source_path = metadata.source_dir
+    # TODO: Detect whether metadata is under project_path or source_path
```

Comment:
See above

Reply:
I think we should make it backward compatible. We haven't done a lot to the packaging flow for `0.19.0` so I'd prefer if we keep it working.

Reply:
This is still pending right? Otherwise happy to approve.

Reply:
Done!